src/test/java/org/folio/circulation/rules/Text2DroolsTest.java

```diff
@@ -44,7 +45,7 @@ public void headerFallbackPolicy() {
     String droolsText = Text2Drools.convert(HEADER);
     Drools drools = new Drools(droolsText);
     assertThat(drools.loanPolicy(params("foo", "bar", "biz", "shelf"),
-      createLocation(FIRST_INSTITUTION_ID, FIRST_LIBRARY_ID, FIRST_CAMPUS_ID)),
+      createLocation(FIRST_INSTITUTION_ID, FIRST_LIBRARY_ID, FIRST_CAMPUS_ID)).getPolicyId(),
```

Comment:
Can these tests also be expanded to include checks that the correct rule conditions are also returned?

Reply:
I don't think that it makes sense, because there are no rule conditions here and rule conditions entity has default state
![image](https://user-images.githubusercontent.com/53909129/72141055-bac9f800-339a-11ea-9b94-16415418c6ff.png)